Meshtastic/Resources/DeviceHardware.json

```diff
@@ -318,5 +318,53 @@
 	"architecture": "esp32-s3",
 	"activelySupported": true,
 	"displayName": "unPhone"
+  },
+  {
+	"hwModel": 48,
+	"hwModelSlug": "HELTEC_WIRELESS_TRACKER",

```

Comment:
@thebentern any idea why the api is returning the same device hwModel multiple times

Reply:
Same actual hardware model but different variant with SPI based touch screens, eink, etc.